### ISSUE TITLE
Smart spacing (name?) does not work when "Hide text under icons" is enabled

### ISSUE THREAD

**Issue by ReyLeonardAmorato** [lutris/lutris#4286]
### Bug description

When resizing the lutris window, game thumbnails are spaced evenly to fit the available space. However, enabling the option `Hide text under icons` disables this feature as seen below:

With text
![Screenshot from 2022-05-17 12-53-52](https://user-images.githubusercontent.com/38913802/168733095-5c19db2e-15b2-49b7-9fe5-8ff981a70978.png)

Without text
![Screenshot from 2022-05-17 12-54-32](https://user-images.githubusercontent.com/38913802/168733032-1957cb9f-a60d-400c-956a-d82123e7a453.png)

.

### How to Reproduce

1. Go to 'Preferences'
2. Enable on 'Hide text under icons (requires restart)'
3. Restart Lutris
4. Resize window
5. Thumbnails are no longer spaced evenly

### Expected behavior

Game thumbnails should be spaced evenly with or without text under them.

### Log output

```shell
$ lutris -d
INFO     2022-05-17 13:08:55,601 [startup.init_lutris:173]:Starting Lutris 0.5.10
INFO     2022-05-17 13:08:55,676 [startup.check_driver:65]:Running AMD Mesa driver 22.0.1 on AMD Radeon RX 5700 XT (navi10, LLVM 13.0.1, DRM 3.42, 5.15.0-30-generic) (0x731f)
INFO     2022-05-17 13:08:55,676 [startup.check_driver:77]:GPU: 1002:731F 1002:0B36 (amdgpu drivers)
INFO     2022-05-17 13:08:55,738 [startup.update_runtime:213]:Startup complete
DEBUG    2022-05-17 13:08:55,848 [lutriswindow.update_store:437]:Showing 58 games
```

### System Information

```shell
[System]
OS:              Ubuntu 22.04 Jammy Jellyfish
Arch:            x86_64
Kernel:          5.15.0-30-generic
Desktop:         ubuntu:GNOME
Display Server:  wayland

[CPU]
Vendor:          AuthenticAMD
Model:           AMD Ryzen 5 2600 Six-Core Processor
Physical cores:  6
Logical cores:   12

[Memory]
RAM:             31.3 GB
Swap:            1.7 GB

[Graphics]
Vendor:          AMD
OpenGL Renderer: AMD Radeon RX 5700 XT (navi10, LLVM 13.0.1, DRM 3.42, 5.15.0-30-generic)
OpenGL Version:  4.6 (Compatibility Profile) Mesa 22.0.1
OpenGL Core:     4.6 (Core Profile) Mesa 22.0.1
OpenGL ES:       OpenGL ES 3.2 Mesa 22.0.1
Vulkan:          Supported
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by danieljohnson2** [comment]
I don't think this is actually a defect. The cover-art isn't actually being evenly spaced at all, as far as I can tell. Not exactly.

The effect you are seeing is caused by the layout trying to accommodate the text, which is too wide to fit under the art. It expands the spacing to try to fit the text, and can't find enough space so it word-wraps anyway. It distributes the extra space evenly, so you get this 'justified' effect.

*But* if you view only games with short names, this does not happen and the art winds up left justified, even when the names are visible.

**Comment by ReyLeonardAmorato** [comment]
This feature of auto resizing of the gaps between the thumbnails was only added into lutris a few versions ago, I couldn't remember what version exactly, but it didn't use to do this. I'm merely saying that when texts are gone, that feature is also gone.

Some of the text are short and not wrapped in anyway so I don't think it's about finding space for the text. Please actually try resizing the lutris window with games with and without text to see what I mean. The gaps in between game thumbnails **adaptively** changes according to the available window width whereas if you disable text that adaptive spacing is gone.
